ia-guide/ia-static.md

```diff
@@ -0,0 +1,341 @@
+
+Have you ever been really good at something, and tried to show someone else how to do it?
+It's frustrating when they just don't seem to get it, even though it seems easy to you.
+Science tells us that the more you know about a topic, the less you are able to imagine what it's like to not know it.
+The technical name for this is the Curse of Knowledge, and it can make writing documentation hard.
+
+Writers combat the Curse of Knowledge by putting themselves in the shoes of the reader.
+And while this guide won't teach you to be a professional technical writer, it will definitely help you think about your reader, so that you can write better documentation.
+
+Let's dive into the world of the reader ...
+
+## Don't start yet!
+
+When you're excited about a new project, it's super-tempting to just start writing the docs without thinking too hard about it.
+
+**DON'T DO THIS!**
+
+This guide is designed to get you thinking about your readers in such a way that you'll end up knowing what docs you need to write.
+It won't take very long for you to get through it, and it will save you time in the long run.
+
+![A smiling person wearing a blue outfit is in a very big hole, with the words "I am stuck" in red](images/stuck.svg "Plz help")
+
+When you're done, you should have a good idea of who your main readers are, the kinds of things they are using your docs for, and what kinds of docs they might be expecting.
+This will help you know which templates to use from the Good Docs Project, and give you the basics to start completing them.
+
+If you want to do this using the interactive tool, go here instead: [Interactive IA Guide](don't click here the link does not exist yet).
+The interactive version is the same as this static version, so you can use this version to check back on later on if you need to.
+
+## What are you writing about?
+
+The first thing to do is think about what you're writing about.
+
+Are you writing about something pretty simple, that people can use without having any experience? Like a toaster?
+
+Or are you writing about something a bit complicated, that people can start to use only after they've done a bit of reading, and might need to ask some questions as they go along? More like a car?
+
+Or are you writing about something really tricky? Something that requires training, a whole lot of background knowledge, and possibly a friend to help them the whole way? Like a nuclear reactor?
+
+**Curse of Knowledge alert!!**
+> Try to remember that not everyone knows the same stuff, or learns in the same way.
+> What if your reader doesn't speak the same language, or is a new migrant from a place with a totally different culture?
+> What if they are nine years old?
+> Or 99 years old?
+> What if they have a physical or mental disability?
+> When you are thinking about your readers, make sure you are thinking about everyone, not just people like you.
+
+### Writing about a toaster
```

Comment:
I like that you have these 3 sections to show and discuss the increasingly more complex writing situation.  Might be nice to take the opportunity to introduce the specific types of docs one would expect.